### PR TITLE
add libvips 8.6.1 and update nodejs to v8

### PR DESCRIPTION
**Pull request by gmaliar**
this will upgrade to node 8 which is lts version and also added libvips 8.6.1
there's a new sharp version based on libvips 8.5 so I will try to upgrade to that one as well

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,11 @@ We've also included a Node.js 7.10.0 image.
 ## Supported tags
 
 - [`8.4.6`](https://github.com/TailorBrands/docker-libvips/8.4.6/Dockerfile)
-- [`8.5.8`, `latest`](https://github.com/TailorBrands/docker-libvips/8.5.8/Dockerfile)
+- [`8.5.8`](https://github.com/TailorBrands/docker-libvips/8.5.8/Dockerfile)
```
Comment (nadavshatz) [inline README.md]: this is 8.5.9 now no?

Comment (gmaliar) [reply README.md]: yes my bad

**Review by nadavshatz — APPROVED** [review]
LGTM 👏 
Check where/if you want to change tags/names from 8.5.8 to 8.5.9
Maybe have it as 8.5.x?

**Comment by gmaliar** [comment]
changed accordingly in dockerhub

**Pull request merged**